js/binancetr.js

```diff
@@ -0,0 +1,308 @@
+        //            "msg":"Success",
+        //            "data":{
+        //                "list":[
+        //                    {
+        //                        "type":1,
+        //                        "symbol":"ADA_TRY",
+        //                        "baseAsset":"ADA",
+        //                        "basePrecision":8,
+        //                        "quoteAsset":"TRY",
+        //                        "quotePrecision":8,
+        //                        "filters":[
+        //                            {"filterType":"PRICE_FILTER","minPrice":"0.01000000","maxPrice":"10000.00000000","tickSize":"0.01000000","applyToMarket":false},
+        //                            {"filterType":"PERCENT_PRICE","multiplierUp":5,"multiplierDown":0.2,"avgPriceMins":"5","applyToMarket":false},
+        //                            {"filterType":"LOT_SIZE","minQty":"0.10000000","maxQty":"9222449.00000000","stepSize":"0.10000000","applyToMarket":false},
+        //                            {"filterType":"MIN_NOTIONAL","avgPriceMins":"5","minNotional":"10.00000000","applyToMarket":true},
+        //                            {"filterType":"ICEBERG_PARTS","applyToMarket":false,"limit":"10"},
+        //                            {"filterType":"MARKET_LOT_SIZE","minQty":"0.00000000","maxQty":"191987.27930555","stepSize":"0.00000000","applyToMarket":false},
+        //                            {"filterType":"MAX_NUM_ORDERS","applyToMarket":false},
+        //                            {"filterType":"MAX_NUM_ALGO_ORDERS","applyToMarket":false,"maxNumAlgoOrders":"5"}
+        //                        ],
+        //                        "orderTypes":[
+        //                            "LIMIT",
+        //                            "LIMIT_MAKER",
+        //                            "MARKET",
+        //                            "STOP_LOSS_LIMIT",
+        //                            "TAKE_PROFIT_LIMIT"
+        //                            ],
+        //                        "icebergEnable":1,
+        //                        "ocoEnable":1,
+        //                        "spotTradingEnable":1,
+        //                        "marginTradingEnable":0,
+        //                        "permissions":["SPOT"]
+        //                    },
+        //                ]
+        //            }
+        //        }
+        //
+        const result = [];
+        const data = this.safeValue (response, 'data', {});
+        const list = this.safeValue (data, 'list', []);
+        for (let i = 0; i < list.length; i++) {
+            const market = list[i];
+            // const type = this.safeInteger(market, 'type'); // Important to see endpoint for this market
+            const id = this.safeString (market, 'symbol');
+            const filters = this.safeValue (market, 'filters');
+            const filter0 = this.safeValue (filters, 0);
+            const filter2 = this.safeValue (filters, 2);
```

Comment:
We should not expect the order of filters here, please, see how it's done in binance.js:

- https://github.com/ccxt/ccxt/blob/master/js/binance.js#L1492
- https://github.com/ccxt/ccxt/blob/master/js/binance.js#L1493
- https://github.com/ccxt/ccxt/blob/master/js/binance.js#L1553

Reply:
Done.